static/app/views/issueList/actions/actionSet.tsx

```diff
@@ -305,19 +298,3 @@ function isActionSupported(
 }
 
 export default ActionSet;
-
-const Wrapper = styled('div')`
```

Comment:
Moved this and the sort dropdown to the parent component so I could conditionally render `<ActionSet />`.